rust/main/agents/relayer/src/server/message_retry.rs

```diff
@@ -96,9 +96,11 @@ mod tests {
         let (addr, mut rx) = setup_test_server();
 
         let client = reqwest::Client::new();
-        let mut message = HyperlaneMessage::default();
-        // Use a random destination domain
-        message.destination = 42;
+        let message = HyperlaneMessage {
+            // Use a random destination domain
+            destination: 42,
+            ..Default::default()
```

Comment:
why does adding `default()` after not override the destination

Reply:
it's dumb rust syntax, i agree intuitively you'd expect it to apply overrides top to bottom but the only way you can do the `..Default::default()` is if it's at the end